wpt-metadata/chromedriver/headful/webdriver/tests/bidi/input/set_files/invalid.py.ini

```diff
@@ -0,0 +1,2 @@
+[invalid.py]
+  expected: TIMEOUT
```

Comment:
This one seems to be irrelevant

Reply:
Auto-updated. Will check one test is failing unexpectedly